frontend/src/components/MiniCal/MiniCal.tsx

```diff
@@ -39,8 +39,17 @@ const MiniCal = () => {
   };
   class CustomInput extends React.Component<any> {
     render() {
+      const isExpanded = this.props.isCalendarOpen;
+      const ariaExpanded = isExpanded ? 'false' : 'true';
+      const ariaControls = isExpanded ? 'calendar-content' : undefined;
+
       return (
-        <button className={styles.customInput} onClick={this.props.onClick}>
+        <button
+          className={styles.customInput}
+          onClick={this.props.onClick}
+          aria-expanded={ariaExpanded}
```

Comment:
Adding the aria-expanded to this causes a situation where when the modal is not expanded it still reads as expanded see below 


https://github.com/cornell-dti/carriage-web/assets/106533812/e60bf226-e28a-4325-b060-fc1937051a90